guide/react-native/screensharing.mdx

```diff
@@ -0,0 +1,192 @@
+---
+sidebar_position: 5
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Screen sharing
+
+Our SDK also allow to stream content of mobile device screen.
+
+## Installation
+
+### Android
+
+On Android devices, screen share feature is available right after SDK is installed in app. No further actions are required.
+
+### iOS
+
+To enable screen share feature on iOS, you need to follow below steps
+
+<Tabs groupId="app-type">
+
+  <TabItem value="expo" label="Expo">
+
+You need to modify `app.json` file and add our plugin:
+
+```json
+{
+  "expo": {
+    ...
+    "plugins": {
+      ...
+      [
+        "@fishjam-cloud/react-native-client",
+        {
+          "ios": {
+            "enableScreensharing": true
+          }
+        }
+      ],
+      ...
+    }
+  }
+}
+```
+
+  </TabItem>
+  <TabItem value="rn" label="Bare workflow">
+
+Configuring screen sharing on iOS is a little complicated.
```

Comment:
**suggestion:** I'd mention that below config is not needed if you use the above plugin as it does all that.